README.md

```diff
@@ -115,6 +115,14 @@ spring:
     name: demo-app
 ```
 
+### OpenTelemetry Collector
+
+The configuration in the application is identical to the Grafana agent. 
+Whenever this documentation refers to "Grafana Agent", the OpenTelemetry Collector configuration is meant as well.
+
+If you the OpenTelemetry Collector, you can also use the recommended [Spring Boot Dashboard](https://grafana.com/grafana/dashboards/18887), which relies on the prometheus naming conventions (e.g. `_seconds` in the metric names).
```

Comment:
If you use/implement? I think you're missing a few words here.

Reply:
If the Spring Boot Dashboard can be used for the Cloud, Agent and Collector, I would break that out into separate section that directs users to the dashboard and maybe tells them a bit about what the dashboard.  Not sure why it is just being mentioned in Collector's section.